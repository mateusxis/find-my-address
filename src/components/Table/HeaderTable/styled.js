import styled from 'styled-components';

export const SHead = styled.tr`
  display: none;

  @media (min-width: 961px) {
    border-radius: 3px;
    padding: 15px 20px;
    display: flex;
    justify-content: space-between;
    margin-bottom: 25px;
    background-color: #95a5a6;
    font-size: 14px;
    text-transform: uppercase;
    letter-spacing: 0.03em;
  }
`;
