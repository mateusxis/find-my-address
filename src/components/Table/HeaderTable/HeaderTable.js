import { SHead } from './styled';

const HeadTable = ({ children }) => {
  return <SHead>{children}</SHead>;
};

export default HeadTable;
