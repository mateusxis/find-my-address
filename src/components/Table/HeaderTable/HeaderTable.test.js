import renderer from 'react-test-renderer';

import HeaderTable from './HeaderTable';

describe('HeaderTable', () => {
  it('should render HeaderTable', () => {
    const tree = renderer
      .create(
        <HeaderTable>
          <td data-label="City" className="c1">
            City
          </td>
        </HeaderTable>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
