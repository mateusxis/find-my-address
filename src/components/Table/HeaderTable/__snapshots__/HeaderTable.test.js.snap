// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderTable should render HeaderTable 1`] = `
<tr
  className="sc-bdvvtL kpQNUU"
>
  <td
    className="c1"
    data-label="City"
  >
    City
  </td>
</tr>
`;
